vendor/woocommerce/action-scheduler/classes/WP_CLI/Action/Get_Command.php

```
@@ -0,0 +1,75 @@
+	/**
+	 * Execute command.
+	 *
+	 * @return void
+	 */
+	public function execute() {
+		$action_id = $this->args[0];
+		$store     = \ActionScheduler::store();
+		$logger    = \ActionScheduler::logger();
+		$action    = $store->fetch_action( $action_id );
+
+		if ( is_a( $action, ActionScheduler_NullAction::class ) ) {
```

PHPStan (2.1.46)

[error] Class Action_Scheduler\WP_CLI\Action\ActionScheduler_NullAction not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)